### PR TITLE
migrate from geoblacklight-schema

### PR DESCRIPTION
**Pull request by drh-stanford**

### CODE REVIEW

**Comment by drh-stanford** [comment]
There's some major changes in the XSL so we'll need to update the test cases...

File: lib/xslt/fgdc2geoBL.xsl [hunk 1]
```diff
@@ -52,42 +63,40 @@
...
-  <xsl:variable name="y2" select="number(metadata/idinfo/spdom/bounding/northbc)"/><!-- N -->
-  <xsl:variable name="y1" select="number(metadata/idinfo/spdom/bounding/southbc)"/><!-- S -->
-
+ <xsl:variable name="lowerCorner">
+     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/westbc)"/>
+     <xsl:text> </xsl:text>
+     <xsl:value-of select="number(metadata/idinfo/spdom/bounding/southbc)"/>
+</xsl:variable>
   
+  <xsl:variable name="x2" select="substring-before($upperCorner/text(), ' ')"/><!-- E -->
```
Comment (mejackreed) [inline lib/xslt/fgdc2geoBL.xsl]: Ln 79 - 82 were pulled out for a specific reason. Using this xpath doesn't work within Nokogiri, so I modified the way the original fgdc2geobl worked.

File: lib/xslt/iso2geoBL.xsl [hunk 2]
```diff
@@ -10,121 +25,126 @@
...
-        <xsl:otherwise>
-          <xsl:value-of select="$uuid"/>
-        </xsl:otherwise>
-         </xsl:choose>
-    </xsl:variable>
+  </xsl:variable>
+  <xsl:variable name="x2" select="substring-before($upperCorner/text(), ' ')"/><!-- E -->
+  <xsl:variable name="x1" select="substring-before($lowerCorner/text(), ' ')"/><!-- W -->
+  <xsl:variable name="y2" select="substring-after($upperCorner/text(), ' ')"/><!-- N -->
+  <xsl:variable name="y1" select="substring-after($lowerCorner/text(), ' ')"/><!-- S -->
```
Comment (mejackreed) [inline lib/xslt/iso2geoBL.xsl]: Similar issue as listed above in fgdc2geoBl. This just doesn't work with Nokogiri and fails tests.

**Comment by drh-stanford** [comment]
@kimdurante what's the 2.0 XSL features in these transforms and can we reduce them into 1.0 XSL?

**Comment by kimdurante** [comment]
I think 1.0 is fine. I think you mentioned changing one or both of them to
2.0 because the 1.0 had been giving you errors.

Also, these namespaces should probably be removed from the fgdc2geoBL.xsl:

  xmlns="http://www.loc.gov/mods/v3"
  xmlns:gco="http://www.isotc211.org/2005/gco"
  xmlns:gmi="http://www.isotc211.org/2005/gmi"
  xmlns:gmd="http://www.isotc211.org/2005/gmd"
  xmlns:gml="http://www.opengis.net/gml"

**Comment by drh-stanford** [comment]
moot

**Pull request closed without merging**